dask/dataframe/io/tests/test_sql.py

```diff
@@ -72,20 +72,49 @@ def test_datetimes():
                                                  for i in range(2, -3, -1)]})
     with tmpfile() as f:
         uri = 'sqlite:///%s' % f
-        df.to_sql('test', uri, index=True, if_exists='replace')
+        df.to_sql('test', uri, index=False, if_exists='replace')
         data = read_sql_table('test', uri, npartitions=2, index_col='b')
         assert data.index.dtype.kind == "M"
         assert data.divisions[0] == df.b.min()
+        d = data.compute()
+        df2 = df.set_index('b')
+        for i in df2.index:
+            assert df2.a[i] == d.a[i]
```

Comment:
In this case they are not equivalent, because the original times were decreasing, but they come back with the smallest ones first.